tutorials/smart-contracts/launch-your-first-project/create-contracts.md

```diff
@@ -1,6 +1,124 @@
 ---
-title: Create Contracts
-description: TODO
+title: Create a Smart Contract
+description: Learn how to write a basic smart contract using just a text editor. This guide covers creating and preparing a contract for deployment on Asset Hub.
 ---
 
-# Create Contracts
\ No newline at end of file
+# Create a Smart Contract
+
+## Introduction
+
+Creating smart contracts is fundamental to blockchain development. While many frameworks and tools are available, understanding how to write a contract from scratch with just a text editor is essential knowledge.
+
+This tutorial will guide you through creating a basic smart contract that can be used with other tutorials for deployment and integration.
+
+## Prerequisites
+
+Before starting, make sure you have:
+
+- A text editor of your choice (VS Code, Sublime Text, etc.)
+- Basic understanding of programming concepts
+- Familiarity with the Solidity programming language syntax. For further references, check the official [Solidity documentation](https://docs.soliditylang.org/en/latest/){target=\_blank}
+
+## Understanding Smart Contract Structure
+
+A Solidity smart contract follows a basic structure that includes:
+
+1. [SPDX license identifier](https://docs.soliditylang.org/en/v0.6.8/layout-of-source-files.html){target=\_blank}
+2. Pragma directive
+3. Contract declaration
+4. State variables
+5. Functions
+
+Let's create a simple storage contract that will:
+
+- Store a number
+- Allow updating the stored number
+- Emit an event when the number changes
+
+## Create the Smart Contract
+
+1. Create a new file named `Storage.sol` in your text editor
+
+2. Add the SPDX license identifier at the top of the file. The Solidity compiler requires this:
```

Comment:
For such a basic tutorial, it might help to explain a little more than normal for the audience we're targeting. Why is it required? What does it do? (although probably intuitive)